mods/windows-11-notification-center-styler.cpp

```diff
@@ -0,0 +1,1815 @@
+// ==WindhawkMod==
+// @id              windows-11-notification-center-styler
+// @name            Windows 11 Notification Center Styler
+// @description     An advanced mod to override style attributes of the Notification Center
+// @version         1.0.0
+// @author          thx1200
+// @github          https://github.com/thx1200
+// @include         ShellExperienceHost.exe
+// @architecture    x86-64
+// @compilerOptions -lcomctl32 -lole32 -loleaut32 -lruntimeobject -Wl,--export-all-symbols
+// ==/WindhawkMod==
+
+// Source code is published under The GNU General Public License v3.0.
+
+// ==WindhawkModReadme==
+/*
+# Windows 11 Notification Center Styler
+
+An advanced mod to override style attributes of the notification center and calendar.
+
+**Note**: This mod requires Windhawk v1.4 or later.
+
+This is adapted from the **Windows 11 Taskbar Styler** mod.  I really did very little.  All credit goes to m417z.
+
+## Examples
+
+### Hide the focus assist section
+**Target**: ActionCenter.FocusSessionControl  
+**Style**: Height=0
+
+### Square the corners of the notification center
+**Target**: Windows.UI.Xaml.Controls.Grid#NotificationCenterGrid  
+**Style**: CornerRadius=0
+
+### Square the corners of the calendar
+**Target**: Windows.UI.Xaml.Controls.Grid#CalendarCenterGrid  
+**Style**: CornerRadius=0
+
+### Square the corners of the quick action center
+**Target**: Windows.UI.Xaml.Controls.Grid#ControlCenterRegion  
+**Style**: CornerRadius=0
+
+### Calendar and notification titlebars: titles on the right, buttons on the left
+**Target**: Windows.UI.Xaml.Controls.Grid#RootContent  
+**Style**: FlowDirection=1
+
+### Add accelerator key (ALT+X) to clear all notifications
+**Target**: Windows.UI.Xaml.Controls.Button#ClearAll  
+**Style**: AccessKey=x
+
+### Add accelerator key (ALT+E) to expand/collapse the calendar
+**Target**: Windows.UI.Xaml.Controls.Button#ExpandCollapseButton  
+**Style**: AccessKey=e
+
+## Advanced styling
+
+The settings have two sections: control styles and resource
+variables. Control styles allow to override styles, such as size and color, for
+the target elements. Resource variables allow to override predefined variables.
+For a more detailed explanation and examples, refer to the sections below.
+
+The notitfication center's XAML resources can help find out which elements and resource
```

Comment:
Typo (notitfication). Also that paragraph is no longer relevant since it links to the start menu XAML files.